soc/arm/nxp_lpc/lpc51u68/Kconfig.defconfig.series

```diff
@@ -0,0 +1,42 @@
+# LPC51U68 series configuration options
+
+# Copyright (c) 2021 metraTec GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_LPC51U68
+
+config SOC_SERIES
+	default "lpc51u68"
+
+config NUM_IRQS
+	# must be >= the highest interrupt number used.
+	default 32
+
+config SOC
+	default "lpc51u68"
+
+config PINMUX_MCUX_LPC
+	default y
+	depends on PINMUX
+
+config GPIO_MCUX_LPC
+	default y
+	depends on GPIO
+
+config UART_MCUX_FLEXCOMM
+	default y
+	depends on SERIAL
+
+config SOC_FLASH_LPC
+	default y
+	depends on FLASH
+
+config PWM_MCUX_SCTIMER
+	default y
+	depends on PWM
+
+config HAS_LPC_ADC
```

Comment:
This should be `ADC_LPC_ADC`, and added in a later commit after that symbol is introduced